test/fixtures/classic-relative-imports/input.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  app: {
+    utils: {
+      'nested.js': '"nested util"',
+      'single.js': '"single util"',
+      'derived-nested.js': 'import NestedUtil from "./nested";',
+      'derived-single.js': 'import SingleUtil from "./single";'
+    },
+    routes: {
+      'application.js': 'import NestedUtil from "../utils/nested"; import SingleUtil from "../utils/single";',
+      'index.js': 'import ApplicationRoute from "./application";',
+      post: {
+        'index.js': 'import ApplicationRoute from "../application";'
+      }
+    }
+  },
+  tests: {
+    unit: {
+      utils: {
+        'nested-test.js': '"nested util test"'
+      },
+      routes: {
+        'application-test.js': 'import ApplicationRoute from "../../../app/routes/application";'
```

Comment:
@andyhot Yes you are correct the `/app` part was a mistake. I've pushed some changes to the 201-created branch today. Please have a look and feel free to copy the same examples. I also added to the tests so ember-data examples (models and adapters). Finally, I updated the tests for relative paths as well, although I never implemented support for relative paths on that branch--only the absolute path tests are passing.

https://github.com/rwjblue/ember-module-migrator/compare/master...201-created:import-test?expand=1